src/components/TopicForm/TopicForm.tsx

```diff
@@ -24,6 +24,12 @@ const sectionTopics = availableTopics.map(topic => {
   return (
     <>
       <Header />
+      <Link to="/saved">
```

Comment:
These buttons need styling. When view them on mobile view, they are too close together and it is difficult to click them.

Reply:
Fixed the margins, but still needs some pretty styling.